### PR TITLE
Clone from source branch for metal3-io.github.io

### PR DESCRIPTION
**Pull request by fmuyassarov**
Clone from `source ` branch of metal3-io.github.io

### CODE REVIEW

**Comment by fmuyassarov** [comment]
> It is not sufficient to change the branch for upstream, you also need to change it for Nordix, or you will push the changes from source onto master, and probably get a conflict

Ah good point!

File: scripts/update-nordix-repos-master.sh [hunk 1]
```diff
@@ -72,12 +72,12 @@ do
   echo "Updating master branch in ${repo}"
   pushd "${repo}"
   BRANCH=$(git rev-parse --abbrev-ref HEAD)
-  git checkout "${UPDATE_BRANCH}"
+  git checkout "${BRANCH}"
```
Comment (maelk) [inline scripts/update-nordix-repos-master.sh]: this needs to be set to source in the metal3 website case. It defaults to master

Comment (fmuyassarov) [reply scripts/update-nordix-repos-master.sh]: If I didn't miss anything, both Nordix and upstream defaults to source branch in case of metal3 website

**Comment by fmuyassarov** [comment]
> Ok, sorry, reviewed too fast

No worries.

**Pull request merged**